src/Utilities/matrix_of_constraints.jl

```diff
@@ -523,6 +526,19 @@ function load_constants(
     return
 end
 
+load_constants(::Vector, ::Any, ::MOI.AbstractVectorSet) = nothing
```

Comment:
can be in another PR but shouldn't `load_constants` be documented or underscored?

Reply:
It already is
https://github.com/jump-dev/MathOptInterface.jl/blob/9ef6efefc4610eb44e4dd7fc1deea2a538b13fa0/src/Utilities/matrix_of_constraints.jl#L175-L192

Reply:
my bad, hadn't seen it!